### PR TITLE
docs: update stackable overview image

### PR DESCRIPTION
**Pull request by xeniape**
This PR updates the overview image for the SDP in the readmes of the operators.

### CODE REVIEW

File: template/.readme/static/borrowed/stackable_overview.png
[the remark this answers is not in the document]
Comment (xeniape) [reply template/.readme/static/borrowed/stackable_overview.png:1]: Came from Fabian :D not sure what his Github handle is if he even has an account. Will link the feedback to him in Slack.

[the remark this answers is not in the document]
Comment (nightkr) [reply template/.readme/static/borrowed/stackable_overview.png:1]: @stackfab, I think?

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply template/.readme/static/borrowed/stackable_overview.png:1]: Semi-relevant, you can have dark and light-mode images in Github markdown: https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/

I think in this case we just want a solid background, but wanted to share that.

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply template/.readme/static/borrowed/stackable_overview.png:1]: Here's a preview of the problem for @stackfab:

image

**Review by stackfab — COMMENTED** [review]
I cannot really change the graphic to anything more dark mode friendlier ATM. Once we move towards DM in 2025, we need solutions for logos in general. I would like to postpone it until then. Minor side note, @sbernauer requested the HDFS image update. Unfortunately the new image exists only in a very low resolution.

**Comment by nightkr** [comment]
Sure you can, @stackfab, just get rid of the rounded corners and it instantly becomes much less jarring, just like the old one:

image

image

Or you could replace the white background with a transparent one, but that would be much easier with access to the original source files instead of dragging things around in Krita for five minutes.

**Comment by sbernauer** [comment]
We can merge as is *for now*, and have a follow-up regarding the graphic in general (modernize, include source, dark mode)

**Comment by stackfab** [comment]
I will provide an slightly updated version until noon (16/12/24)

**Review by nightkr — APPROVED** [review]

**Pull request merged**